pkgs/top-level/all-packages.nix

```diff
@@ -24530,6 +24530,7 @@ with pkgs;
   tgt = callPackage ../tools/networking/tgt { };
 
   lkl = callPackage ../applications/virtualization/lkl { };
+  lklWithFirewall = callPackage ../applications/virtualization/lkl { firewallSupport = true; };
```

Comment:
I think this is a useful addition. Can you also link to an example on how to use this feature to check nftable rules?